packages/litexa-deploy-aws/src/utils/s3Utils.coffee

```diff
@@ -83,7 +83,11 @@ registerFileForUpload = ({ s3Context, fileDir, fileName, language }) ->
   fileName = fileName.replace /\\/g, '/'
   sourceFilePath = "#{fileDir}/#{fileName}"
   s3Context.assetCount += 1
-  s3Key = "#{s3Context.baseLocation}/#{language}/#{fileName}"
+  # S3 won't take the wrong slash, and in any case we want
+  # this key to be the same no matter which platform we come
+  # from
+  canonicalFilename = fileName.replace /\\/g, '/'
```

Comment:
the filename replacement was done earlier in line 83, is this still needed?

Reply:
LOLWUT! It looks like I redundantly patched master for a patch in develop that hasn't made it there yet. My fail. Looking at the function again though, we do have a non-cross-platform friendly path concatenation, so I'll pivot this pull request to address that instead :)

Reply:
Alright, new PR switches to the issue of platform specific concatenation. I sort of want to review this whole data flow to differentiate between local filename reference and constructed URL references. I think everything is right now, but I don't have a consistent windows user to check with.